common/core/overlaybuffer.cpp

```diff
@@ -65,23 +65,40 @@ GpuImage OverlayBuffer::ImportImage(GpuDisplay egl_display) {
   // Note: If eglCreateImageKHR is successful for a EGL_LINUX_DMA_BUF_EXT
   // target, the EGL will take a reference to the dma_buf.
   if (is_yuv_) {
-    const EGLint attr_list_yv12[] = {
```

Comment:
You should fix commit message *is not*

Also, add one Jira and Tests would be u video test u just added. You can list it here.